README.md

```diff
@@ -14,6 +14,8 @@ A style guide for the [HTML Template Language](https://docs.adobe.com/docs/en/ht
   - [1.1](#1.1) <a name='1.1'></a> **Avoid inline JavaScript or CSS.**
   
   In order to encourage keeping a clean separation of concerns, HTL has by design some limitations for inline JavaScript or CSS. First, because HTL doesn't parse JavaScript or CSS, and therefore cannot automatically define the corresponding escaping, all expressions written there must provide an explicit `context` option. Then, because the HTML grammar ignores elements located inside a `<script>` or `<style>` elements, no block statement can be used within them.
+ 
+Unless you are using HTML mail for campaign, in which case, inline styles can be set by SASS compilers.
```

Comment:
It is not 100% clear what the “unless” refers to here. I guess this refers to inline styling and not to ignored block elements. Therefore I recommend to move this sentence to another place

Reply:
the "unless" it was a suggestion now that Campaign is more relevant in html/htl components.  The use of inline is a way to make mails compatible with clients and that is why it is not included as external css.

Reply:
Sorry, but I don't think that this is clear enough and that we should even mention that in the guidelines. It is kind of obvious that in some edge cases you need to deviate from the best practices.